config/markdownlint-cli/.markdownlintignore

```diff
@@ -0,0 +1,8 @@
+# https://github.com/igorshubovych/markdownlint-cli?tab=readme-ov-file#ignoring-files
+vendor/
+node_modules/
+
```

Comment:
```suggestion
LICENSE.md
```

Er den ikke nogle steder? kan ikke huske om det er det den hedder.

Reply:
Filename convention varies between `LICENSE`, `LICENSE.md` and even `LICENSE.txt`

Reply:
We only check `**/*.md` and don't need to ignore other file formats.